src/services/categoryServices.js

```diff
@@ -0,0 +1,192 @@
+const Category = require("../models/categories")
+const Product =require("../models/products")
+const config = require("../config/config.js")
+const logger = require('../logger/logger')
+
+class categoryService {
+    /**
+         * create Category
+         * @param {object} requestParams
+         * @returns {object}
+         * @author khushbuw
+         */
+    static async createCategory(userRole,requestParams) {
+        try {
+            if ((userRole == 'Admin') || (userRole == 'admin')) {
+                if (!(requestParams.name) ) {
+                    return ({ error: config.emptyFields });
+                }
+                const createdCategory = await Category.create(requestParams);
+                
+                logger.info({ message: "Category created", info: createdCategory });
+                return ({
+                    success: config.recordCreated,
+                });
+            } else {
+                return ({ error: config.notAllowedCreate });
+            }
+        } catch (err) {
+            logger.error({ error_message: err.message });
+            return ({ error_message: err.message });
+        }
+    }
+
+
+
+    /**
+     * Category list
+     * @param {object} requestParams
+     * @returns {object}
+     * @author khushbuw
+     */
+    static async listCategory(requestQueries) {
+        try {
+            var sortObject = {};
+            const limit = requestQueries.limit
+            const page = requestQueries.page
+            const sort = requestQueries.sort
+            const sortingMethod = requestQueries.sortingMethod
+            var key = requestQueries.key
+            if (!key) {
+                key = ''
+            }
+            var categoryData;
+            var orderBy;
+            (sortingMethod === 'desc') ? orderBy = '-1' : orderBy = '1'
+            var skip;
+            (page <= 1) ? skip = 0 : skip = (page - 1) * limit
```

Comment:
Please refactor this code like this
`const orderBy =  (sortingMethod === 'desc') ? -1 : 1;`